### PR TITLE
Machine collection

### PR DESCRIPTION
**Pull request by philjdelong**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- Create worker to update reading collections within a specified timeframe. This should be automatically refreshed upon visiting '/readinglist'
## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/models/reading_collection.rb [hunk 1]
```diff
@@ -5,9 +5,10 @@ class ReadingCollection < ApplicationRecord
   validates :name, :user_id, presence: { presence: true }
   validates :slug, presence: true, format: /\A[0-9a-z\-_]*\z/, uniqueness: { scope: :user_id }
   before_validation :set_slug
+  acts_as_taggable_on :tags
 
   def get_articles
-    Article.where("created_at >= ?", 1.week.ago).
+    Article.where("created_at >= ?", 5.minutes.ago).
```
Comment (joel-oe-lacey) [inline app/models/reading_collection.rb:11]: Real time news, I like this.

File: app/workers/reading_collections/refresh_reading_collection_worker.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module ReadingCollections
+  class RefreshReadingCollectionWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
```
Comment (joel-oe-lacey) [inline app/workers/reading_collections/refresh_reading_collection_worker.rb:5]: What is the priority doing here?

File: spec/workers/reading_collections/refresh_collection_worker_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# require "rails_helper"
+
+# RSpec.describe ReadingCollections::RefreshReadingCollectionWorker, type: :worker do
```
Comment (mintona) [inline spec/workers/reading_collections/refresh_collection_worker_spec.rb:3]: Bug fix for tomorrow!

**Review by joel-oe-lacey — APPROVED** [review]

**Review by mintona — APPROVED** [review]
Paired w/ Phil on this. Great job!

**Pull request merged**